lemur/manage.py

```diff
@@ -112,6 +112,7 @@
 LEMUR_DEFAULT_LOCATION = ''
 LEMUR_DEFAULT_ORGANIZATION = ''
 LEMUR_DEFAULT_ORGANIZATIONAL_UNIT = ''
+LEMUR_DEFAULT_ROTATION = False
```

Comment:
I like that we are not requiring this new variable, and just default to `False`

https://github.com/Netflix/lemur/blob/3fbcbc27174622b6075b492eccc1a42530afc5f0/lemur/manage.py#L61